<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PMS 数据库迁移工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="currDBVersionLabel">
       <property name="text">
        <string>当前数据库版本：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currDBVersionLineEdit">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="transToDBVersionLabel">
       <property name="text">
        <string>迁移到版本：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="transToDBVersionLineEdit">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="inputDBLabel">
       <property name="text">
        <string>输入数据库：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputDBPath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputSelectButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="outputDBLabel">
       <property name="text">
        <string>输出数据库：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputDBPath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputSelectButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
